src/ContainerAppHelper.ts

```diff
@@ -0,0 +1,434 @@
+     * @param imageToDeploy - the name of the runnable application image that the Container App will be based from
+     * @param optionalCmdArgs - a set of optional command line arguments
+     */
+    public async createContainerApp(
+        containerAppName: string,
+        resourceGroup: string,
+        environment: string,
+        imageToDeploy: string,
+        optionalCmdArgs: string[]) {
+        toolHelper.writeDebug(`Attempting to create Container App with name "${containerAppName}" in resource group "${resourceGroup}" based from image "${imageToDeploy}"`);
+        try {
+            let command = `az containerapp create -n ${containerAppName} -g ${resourceGroup} -i ${imageToDeploy} --environment ${environment}`;
+            optionalCmdArgs.forEach(function (val: string) {
+                command += ` ${val}`;
+            });
+            await util.executeAndThrowIfError(command);
+        } catch (err) {
+            toolHelper.writeError(err.message);
+            throw err;
+        }
+    }
+
+    /**
+     * Creates an Azure Container App based from a YAML configuration file.
+     * @param containerAppName - the name of the Container App
+     * @param resourceGroup - the resource group that the Container App is found in
+     * @param yamlConfigPath - the path to the YAML configuration file that the Container App properties will be based from
+     */
+    public async createContainerAppFromYaml(
+        containerAppName: string,
+        resourceGroup: string,
+        yamlConfigPath: string) {
+        toolHelper.writeDebug(`Attempting to create Container App with name "${containerAppName}" in resource group "${resourceGroup}" from provided YAML "${yamlConfigPath}"`);
+        try {
+            let command = `az containerapp create -n ${containerAppName} -g ${resourceGroup} --yaml ${yamlConfigPath}`;
+            await util.executeAndThrowIfError(command);
+        } catch (err) {
+            toolHelper.writeError(err.message);
+            throw err;
+        }
+    }
+
+    /**
+     * Updates an existing Azure Container App based from an image that was previously built.
+     * @param containerAppName - the name of the existing Container App
+     * @param resourceGroup - the resource group that the existing Container App is found in
+     * @param imageToDeploy - the name of the runnable application image that the Container App will be based from
+     * @param optionalCmdArgs - a set of optional command line arguments
+     */
+    public async updateContainerApp(
+        containerAppName: string,
+        resourceGroup: string,
+        imageToDeploy: string,
+        optionalCmdArgs: string[]) {
+        toolHelper.writeDebug(`Attempting to update Container App with name "${containerAppName}" in resource group "${resourceGroup}" based from image "${imageToDeploy}"`);
+        try {
+            let command = `az containerapp update -n ${containerAppName} -g ${resourceGroup} -i ${imageToDeploy}`;
```

Comment:
We need to add `--output none` to the end of this command call per the latest changes made in `v1`

Reply:
Updated.